lib/src/base-styles/_nx-variables.scss

```
@@ -65,9 +65,14 @@
                             0 1px 2px 0 rgba(91, 95, 141, 0.2),
                             0 1px 1px 0 rgba(124, 128, 179, 0.2);
 
+  // NxDrawer
+  --nx-box-shadow-drawer: -34px 0 50px -32px hsla(0, 0%, 50%, 0.5);
+
   @include nx-dark-mode-helpers.dark-mode {
     --nx-border-grid: var(--nx-border-default);
     --nx-border-width: 2px;
+    // NxDrawer
+    --nx-box-shadow-drawer: -34px 0 50px -32px hsla(231, 41%, 10%, 0.75);
```

```suggestion
    --nx-box-shadow-drawer: -34px 0 50px -32px hsla(var(--nx-swatch-indigo-hs), 10%, 0.75);
```
Could do the light mode one similarly, though it's less necessary with grey

---

>Could do the light mode one similarly, though it's less necessary with grey

Updated here, including the light mode (seems nicer to have the light/dark mode variables defined the same way) [7e25ef6](https://github.com/sonatype/sonatype-react-shared-components/pull/1097/commits/7e25ef66b14f6de8c1d93ad316da5864a951b0cb)